Xamarin.Forms.Core/IEmbeddedFontLoader.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+namespace Xamarin.Forms
+{
+	public interface IEmbeddedFontLoader
```

Comment:
is this meant to be consumed by platform only ? in that case hide it from intelisense